airbyte-webapp/src/hooks/useQuery.tsx

```diff
@@ -0,0 +1,8 @@
+import queryString from "query-string";
+import { useMemo } from "react";
+import { useLocation } from "react-router-dom";
+
+export const useQuery = () => {
```

Comment:
In order to avoid casting, you could use generics here:

```
export const useQuery = <T>() => {
```

Reply:
fixed